xayn-ai/src/coi/config.rs

```diff
@@ -207,3 +208,61 @@ impl Config {
         self.kpe.penalty.len()
     }
 }
+
+mod serde_horizon_days {
```

Comment:
It doesn't seem like this is specific to horizon, so it could also be called `serde_days` or something similar, and live in a more generic location.